widgetry/src/runner.rs

```diff
@@ -353,6 +353,7 @@ pub fn run<
 
     let mut running = true;
     let mut last_update = Instant::now();
+    let mut previous_left_click_at = Instant::now();
```

Comment:
```suggestion
    // The user will not manage to click immediately after the window opens, so this initial value is simpler than an `Option<Instant>`
    let mut previous_left_click_at = Instant::now();
```
(I had to think about it, so a comment could be nice)